drivers/include/drivers/OSPI.h

```diff
@@ -229,8 +229,6 @@ class OSPI : private NonCopyable<OSPI> {
 
     ospi_t _ospi;
 
-    bool acquire(void);
```

Comment:
This was declared but not defined anywhere?

Reply:
Yes it appears so